package.json

```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "setup-dbs": "psql -f ./db/setup.sql",
     "seed": "node ./db/seeds/run-seed.js",
-    "test": "jest",
+    "test": "PGDATABASE=nc_news_test jest",
```

Comment:
we shouldn't need to add this to our test script if our `.env` files are running correctly and we add our `beforeEach` and `afterAll` to the test file 😊